apis/constants.ts

```diff
@@ -0,0 +1,10 @@
+const VERSION = "v1";
+const BASE_URL = "http://localhost:3000/api";
+export const API_URL = `${BASE_URL}/${VERSION}`;
+export const DEFAULT_TIMEOUT = 10000;
+export const API_URI = Object.freeze({
+  USER: "/user",
```

Comment:
단일 리소스가 아니니 `/users`로 수정해야합니다. 
#5 요구사항을 확인해주세요.

```suggestion
  USERS: "/users"
```